.github/workflows/rust.yml

```diff
@@ -52,14 +52,15 @@ jobs:
         with:
           files: target/nextest/default/junit.xml
 
-  proteus-test:
+  all-features:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v5
       - uses: ./.github/actions/setup-and-cache-rust
       - uses: taiki-e/install-action@nextest
-      - name: "test corecrypto's proteus implementation"
-        run: cargo nextest run --locked --verbose --features proteus,proteus-keystore proteus
```

Comment:
Note to reviewers: the deleted tests are a strict subset of the new test configuration.